packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -83,7 +84,14 @@ class TestCommand extends FastFlutterCommand {
         abbr: 'j',
         defaultsTo: math.max<int>(1, platform.numberOfProcessors - 2).toString(),
         help: 'The number of concurrent test processes to run.',
-        valueHelp: 'jobs');
+        valueHelp: 'jobs'
+      )
+      ..addFlag('no-assets',
```

Comment:
Added a flag.
Two thing concerns me here.
1. Should we build the assets by default?
2, When no-assets, should we still try to see if there is test assets built in previous test and use it if it exist?

Reply:
My current implementation is yes for both

Reply:
my two general concerns:

1. Test startup speed - does building the asset bundle slow it down measurably? Try `time flutter test <<something_test.dart>>` in the gallery and compare.
2. Users that are already mocking assets. Maybe `--no-assets` should also not set the env variable

Reply:
I didn't see too much difference. maybe 1 or 2 seconds slower out of 2mins. But I do want to point out one thing. The existing test that involve assets load are currently passed because of our fake async. The assets load never finished without my fix. Otherwise they will all fail. This might add some more running time